packages/components/organisms/f-checkout/src/components/_tests/Selector.test.js

```diff
@@ -496,6 +496,51 @@ describe('Selector', () => {
                     expect(wrapper.vm.selectedAvailableFulfilmentTime).toBe('');
                 });
             });
+
+            describe('when a future fulfilment time has been selected', () => {
+                // eslint-disable-next-line prefer-destructuring
+                const selectedTime = fulfilmentTimes[1];
+                const asapTime = fulfilmentTimes[0].from;
+                const laterTime = fulfilmentTimes[1].from;
```

Comment:
Can we place these inside a `beforeEach`? If someone changes `fulfilmentTimes` within a test, it will have an impact on this.